src/get-coverage.ts

```diff
@@ -1,19 +1,23 @@
 import * as lcov from 'lcov-parse';
 import { promisify } from 'util';
 
+import throwError from './lib/error-handling';
 import { CommentData } from './types';
 
 const lcovParse = promisify(lcov);
 
-const getCoverage = async (path: string): Promise<CommentData> => {
-  try {
-    const commentData = {
-      lines: { hit: 0, found: 0, percent: 0, diff: 0 },
-      functions: { hit: 0, found: 0, percent: 0, diff: 0 },
-      branches: { hit: 0, found: 0, percent: 0, diff: 0 },
-    };
-    const coverageData = await lcovParse(path);
+const getCoverage = async (path: string): Promise<CommentData | void> => {
+  const commentData = {
+    lines: { hit: 0, found: 0, percent: 0, diff: 0 },
+    functions: { hit: 0, found: 0, percent: 0, diff: 0 },
+    branches: { hit: 0, found: 0, percent: 0, diff: 0 },
+  };
+
+  let coverageData;
```

Comment:
You don't seem to access this outside the try block so you can just create this as a const in there